06_gpu_and_ml/llm-serving/sgl_vlm.py

```diff
@@ -80,19 +80,18 @@ def download_model():
 
 vlm_image = (
     modal.Image.from_registry(f"nvidia/cuda:{tag}", add_python="3.11")
```

Comment:
nit: could we add `.entrypoint([])` here to silence the NVIDIA container entrypoint logging?

it creates messages like [this one](https://github.com/modal-labs/modal-examples/actions/runs/16359157935/job/46223778864?pr=1284#step:4:40) during all of the install steps, since they don't have GPU drivers exposed to the container